app/models/question_answer.rb

```diff
@@ -0,0 +1,4 @@
+class QuestionAnswer < ActiveRecord::Base
+  belongs_to :question
```

Comment:
We might want to consider either storing the question text along with the answer (so that the answer maintains it's contextual information in case the question is edited/deleted) or prevent the user from deleting questions but merely removing them from worksheets (so that the integrity of the question_id is kept)